refinery/ui/source/js/data-set-import/controllers/isa-tab-import.js

```diff
@@ -82,8 +84,12 @@
       formData.append('identity_id', AWS.config.credentials.identityId);
     }
 
+    var queryParams = {};
+    if (this.isMetaDataRevision) {
+      queryParams = { data_set_uuid: this.dataSetUUID };
+    }
```

Comment:
Maybe just use `this.dataSetUUID` directly, instead of introducing the `this.isMetaDataRevision`? Eh.

Reply:
Never mind, I think you're right that it makes it more clear.